### PR TITLE
Fix font size preference not updating in preference dialog 8386

### PR DESCRIPTION
**Pull request by tomazari**
Fixes: [Issue #8386](https://github.com/JabRef/jabref/issues/8386)

Issue caused by ThemeManager using an empty string in order to set the font size to the default setting. Resolved by providing the default value instead of relying on the css file default.

May also fix: [Issue #9279](https://github.com/JabRef/jabref/issues/9279)

- [x] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

File: src/main/java/org/jabref/gui/theme/ThemeManager.java
Comment (Siedlerchr) [inline src/main/java/org/jabref/gui/theme/ThemeManager.java:218]: This is ugly. Add a method in the appearence preferences, call it something like getDefaultFontSize() and pass the default value to it.

**Review by Siedlerchr — CHANGES_REQUESTED** [review]

File: src/main/java/org/jabref/gui/theme/ThemeManager.java
Comment (tomazari) [reply src/main/java/org/jabref/gui/theme/ThemeManager.java]: @Siedlerchr Agreed, it is not an ideal solution. 
I have added the `getDefaultFontSize()` method to the `AppearancePreferences` class. The default value is passed to the class in the constructor. The ThemeManager now uses the getter method to set the default font size :)

Comment (Siedlerchr) [reply src/main/java/org/jabref/gui/theme/ThemeManager.java]: Thanks this looks much better now!

**Review by Siedlerchr — APPROVED** [review]
From my side it looks good now! 
A second maintainer now needs to do another review

**Comment by koppor** [comment]
I can confirm that https://github.com/JabRef/jabref/issues/8386 and https://github.com/JabRef/jabref/issues/9279 are fixed.

**Review by koppor — APPROVED** [review]

**Pull request merged**
